drupal/web/sites/default/settings.php

```diff
@@ -116,7 +116,7 @@
  */
 if (!drupal_installation_attempted() && extension_loaded('memcached') && class_exists('Memcached')) {
   // Define memcache settings only if memcache module enabled.
-  if (class_exists(MemcacheBackendFactory::class)) {
+  if (strpos(file_get_contents(getcwd().'/../sync/core.extension.yml'), 'memcache: 0') !== FALSE) {
```

Comment:
its a bit too specific.
it might be "memcache: 1001" as well.
try preg_match(), which is slightly more expensive.